package.json

```diff
@@ -60,6 +60,7 @@
     "v-calendar": "^3.1.2",
     "vue-chart-3": "^3.1.8",
     "vue-chartjs": "^5.3.1",
+    "vue-echarts": "^7.0.3",
```

Comment:
@DezengKong are we still using vue-chart-3 and those other libraries? if not we should clean them up